### DISCUSSION TITLE
can i change location

### DISCUSSION THREAD

**Discussion by faraway20** [Chia-Network/chia-blockchain#6271] [Ideas]
hello, maybe my question stupid, but i want to know i understanding ok or not. 
1. Starting plotting on my gaming PC.
2. Choose in system, completed plots sent to external hdd. 
3. When external hdd is full i disconnect him from PC
4. I install in raspberry pi utuntu + chia software
5. Connect to pi my external hdd with plots
6. Connect to chia with my game pc credencials
7. Show to chia software where in external hdd is plots
8. System sinchronize and start farming.

I understand correct or no? thx for answers

**Comment by XIII39** [comment]
Your raspberry pi is the farm, correct?
As long as you are using the correct wallet (mnemonic) so that the raspberry pi has the correct Farm Key and Public Pool Key, I don't see any issues with your setup.

**Reply by faraway20** [reply]
yes, pi just farm, thx for answer. 
how you think, pi4b 8gb is enought for farm?

**Reply by XIII39** [reply]
I don't personally have a ton of experience with raspberry pi myself, I would assume it would be good enough to farm on, farming doesn't really use too many resources but, maybe someone with more experience with raspberry pi has a better idea.
